package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "financial-modeling-prep-mcp-server",
-  "version": "2.0.0",
+  "version": "3.0.0",
```

Comment:
It's a breaking change? (by default you wouldn't get all tools?)

Reply:
it is not, I managed to not break legacy flow.
this is an old change - another good catch